audit-appendix.html

```diff
@@ -96,7 +100,8 @@ <h3>Ingest</h3>
 
     <section>
       <h3>Replication</h3>
-      <p>The Replication event represents the distribution of a filegroup or a file to a single location within a DDP.</p>
+      <p>The Replication event represents the distribution of a filegroup or a file to a single location within a DDP. This should
+      occur for each copy of the filegroup regardless of whether or not it is the primary copy or not.</p>
```

Comment:
Could probably leave off the final "or not" here. The current "whether or not it is the primary copy or not" sounds odd.

Reply:
Agreed. Updated to fix the wording.